### ISSUE TITLE
Deprecation of Emoji functionality September 1, 2024

### ISSUE THREAD

**Issue by tyzbit** [tyzbit/go-discord-archiver#38]
## TL;DR: Archive.org bot will no longer react when the 🏛️ emoji is added to a message after September 1, 2024. Check `/help` for how to get all the same functionality of the bot in better ways.

When this bot was created, it was much more common for bots to use already-existing Discord functionality for control (such as listening for a regular message that says `?8ball` or looking for an emoji added to a message. Now, Discord has much better support for bots such as context menus and slash commands that are not regular messages.

This bot is now added to 100 servers and Discord requires that I add justification for the MessageContent privileged intent. This gives my bot the privilege to read every message in every channel that it's in. The bot only uses it to look for emojis (you can [see for yourself](https://github.com/tyzbit/go-discord-archiver/blob/c7009082f9ec4db33bd71eff7cb231a8bdb0e96c/bot/discord_handlers.go#L101)) but it means that the bot is allowed to see all messages. This isn't good for privacy, I don't want that permission if I don't need it and it turns out, with the better tools built into Discord today, I don't. The context menu actions only give the bot the message that was interacted with, the bare minimum it needs to find URLs in the message and respond.

Discord did not approve my request for privileged MessageContent intent on the grounds that I have the context menu functionality and therefore do not need to see all messages and I agree with their decision. I don't want to change or remove anything without justification but now the old functionality is preventing the bot from being added to any more servers as Discord requires bots in more than 100 servers be reviewed and approved with justification why they need the privileged intents they require.

Removing functionality is the last thing I want to do but if I want this bot to be useful in more servers, it has to be done. Thank you in advance for your understanding.
